### ISSUE TITLE
Doesn't install

### ISSUE THREAD

**Issue by lieszkol** [nsrosenqvist/october-plugin_baguettegallery#2]
I've tried adding your plugin to my site but even though it seems to install, it doesn't show up in my plugins list and no component is created. Perhaps because the plugin it depends on - Responsive Images - no longer seems to exist?

**Comment by nsrosenqvist** [comment]
Yep, that's the issue. I haven't had time to update the plugin. If you'd like to contribute the changes required to skip the dependency on Responsive Images then they are most welcome.
